core/views/files.py

```diff
@@ -71,17 +66,21 @@ def send_file(
         # so please do not mess with this.
         response = HttpResponse(status=200)
         response["Content-Type"] = ""
+        response["Content-Disposition"] = f'inline; filename="{quote(name)}"'
         response["X-Accel-Redirect"] = quote(urljoin(settings.MEDIA_URL, name))
         return response
 
+    filepath = settings.MEDIA_ROOT / name
```

Comment:
Shouldn't we also keep this for the debug server ?

Reply:
Justement, là c'est que pour le serveur debug

Reply:
Ah, oui, effectivement, après c'est nginx qui gère le not found